src/ui/common/src/reducers/workflow.ts

```diff
@@ -497,15 +496,9 @@ export const handleGetSelectDagPosition = createAsyncThunk<
         layoutOptions: {
           'elk.algorithm': 'layered',
           'elk.direction': 'RIGHT',
-          'elk.alignment': 'CENTER',
-          'elk.spacing.nodeNode': '80',
-          'elk.layered.spacing.nodeNodeBetweenLayers': '80',
-          // https://www.eclipse.org/elk/reference/options/org-eclipse-elk-layered-nodePlacement-strategy.html
-          'nodePlacement.strategy': 'NETWORK_SIMPLEX',
-          'org.eclipse.elk.edgeRouting': 'SPLINES',
-          //https://www.eclipse.org/elk/reference/options/org-eclipse-elk-layered-nodePlacement-strategy.html
-          'crossingMinimization.strategy': 'INTERACTIVE',
-          'crossingMinimization.forceNodeModelOrder': true,
```

Comment:
Removing this allows ordering of nodes to not matter when rendering the graph. This was what caused the 'x' to show in one of the churn workflows, but not the other.